.azure-pipelines.yml

```diff
@@ -160,47 +163,54 @@ jobs:
       displayName: Set tag name to "master"
       condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
 
+    - bash: echo "##vso[task.setvariable variable=basename;]wasmtime-$(tagName)-x86_64-windows"
+      displayName: Configure basename var
+      condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'))
+    - bash: echo "##vso[task.setvariable variable=basename;]wasmtime-$(tagName)-x86_64-apple"
+      displayName: Configure basename var
+      condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))
+    - bash: echo "##vso[task.setvariable variable=basename;]wasmtime-$(tagName)-x86_64-linux"
+      displayName: Configure basename var
+      condition: and(succeeded(), eq( variables['Agent.OS'], 'Linux' ))
+
     - bash: |
-        mkdir $BUILD_BINARIESDIRECTORY/bundle
-        cp target/release/{wasmtime,wasm2obj}.exe $BUILD_BINARIESDIRECTORY/bundle
-      displayName: Copy binaries (Win)
-      condition: and(succeeded(), eq( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
-    - bash: |
-        mkdir $BUILD_BINARIESDIRECTORY/bundle
-        cp target/release/{wasmtime,wasm2obj} $BUILD_BINARIESDIRECTORY/bundle
-      displayName: Copy binaries (*nix)
-      condition: and(succeeded(), ne( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
+        set -e
+        mkdir -p $BUILD_BINARIESDIRECTORY/$BASENAME
+        if [ "$AGENT_OS" = "windows" ]; then
+          ext=.exe
+        fi
+        cp LICENSE README.md target/release/{wasmtime,wasm2obj}$ext $BUILD_BINARIESDIRECTORY/$BASENAME
+      displayName: Copy binaries
     - task: ArchiveFiles@2
       inputs:
-        rootFolderOrFile: $(Build.BinariesDirectory)/bundle
-        includeRootFolder: false
+        #rootFolderOrFile: $(Build.BinariesDirectory)
+        #includeRootFolder: false
```

Comment:
I think this needs to be reactivated to remove the spurious `b/` folder in the archives that I [mentioned before](https://github.com/CraneStation/wasmtime/pull/244#issuecomment-518409471).